azure-stack/hci/manage/arc-extension-management.md

```diff
@@ -43,12 +43,57 @@ You can install extensions from the **Capabilities** tab for your Azure Stack HC
 
 When you install an extension in the Azure portal, it's a cluster-aware operation. The extension is installed on all servers of the cluster. If you add more servers to your cluster, all the extensions installed on your cluster are automatically added to the new servers.
 
+
+
 ## Check the extension status
+# [Azure portal](#tab/azureportal)
+After you install an extension, check the status of the extension on each of your nodes. To check the extension status, go to the **Extensions menu page** and view the **status column** of the grid.
 
 You can check the status of an extension on each server from the **Extensions** page by viewing the **status** column of the grid.
 
+
 :::image type="content" source="media/arc-extension-management/arc-extension-status-view.png" alt-text="Screenshot of the different extension statuses in the Azure portal." lightbox="media/arc-extension-management/arc-extension-status-view.png":::
 
+# [Azure CLI](#tab/azurecli)
+Follow these steps to check if the extensions are installed using the Azure CLI.
+
+Azure CLI is available to install in Windows, macOS and Linux environments. It can also be run in a Docker container and Azure Cloud Shell. 
+
+This document details how to use Bash in Azure Cloud Shell. Launch [Azure Cloud Shell]](https://shell.azure.com/). For more information, refer [Quickstart for Azure Cloud Shell](https://learn.microsoft.com/azure/cloud-shell/quickstart)
```

Comment:
Use a relative link instead of an absolute link. Remove everything in the parenthesis before /azure/cloud-shell/quickstart. 

Meaning the relative link should look like (/azure/cloud-shell/quickstart)